src/images/images.service.ts

```diff
@@ -1,7 +1,28 @@
-import { Injectable } from '@nestjs/common';
-import { ImagesRepository } from './repository/images.repository';
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { BoardService } from '../board/board.service';
+import { UploadThumbnailDto } from './dto/upload-thumbnail.dto';
+import { UpdateBoardDto } from '../board/dto/update-board.dto';
 
 @Injectable()
 export class ImagesService {
-  constructor(private readonly imagesRepository: ImagesRepository) {}
+  constructor(private readonly boardService: BoardService) {}
+
+  uploadFile(files: { files: Express.MulterS3.File[] }) {
+    if (!files) {
+      throw new BadRequestException('no file exist');
+    }
+    const name = files.files.map((file) => {
+      return file.key.split('/').at(-1);
+    });
+    return { name };
```

Comment:
```suggestion
    return { names: files.map(({ key }) => key.split('/').at(-1)) };
```
이런 형태는 어떻게 생각하시나요? 위에서 말씀드린 제가 생각하는 클린코드 관점에서는 이렇게 수정할 것 같은데 사실 클린코드가 정답은 없는 부분이라 형이 코멘트를 바라볼 때 긍정적이면 적용하면 되고 크게 와닿지 않으면 그대로 올려도 좋습니다!